include/valijson/constraints/concrete_constraints.hpp

```diff
@@ -915,6 +915,8 @@ class PolyConstraint : public Constraint
             throw;
         }
 #endif
+
+	(void)freeFn;
     }
```

Comment:
It looks like this change may only be required when `VALIJSON_USE_EXCEPTIONS` is not set. Perhaps this would capture that more obviously:
```suggestion
        }
#else
        // pretend to use freeFn to avoid warning in GCC 8.3
        (void)freeFn;
#endif
    }
```